app/src/main/java/com/example/myspacexdemoapp/ui/mappers/LaunchUIMapper.kt

```diff
@@ -156,3 +163,37 @@ class LaunchUIMapper(private val launchData: LaunchData) {
         }
     }
 }
+
+fun HomeData.toTimerUIList(): MutableList<StartUIModel> {
+    val nextLaunch = this.launchData
+    val rocketsEfficiency = this.rockets
+    val timeToLaunch = nextLaunch.mission.date
+    val dataset: MutableList<StartUIModel> = mutableListOf()
+    dataset.add(
+        0,
+        StartUIModel.Timer(
+            name = nextLaunch.mission.name,
+            days = timeToLaunch.day.toString(),
+            hours = timeToLaunch.hours.toString(),
+            minutes = timeToLaunch.minutes.toString(),
+            seconds = timeToLaunch.seconds.toString()
+        )
+    )
+    dataset.add(
+        1,
+        StartUIModel.Launches(
+            successful = rocketsEfficiency.efficiency,
+            total = rocketsEfficiency.total,
+            efficiency = "",
+            toLaunches = ""
+        )
+    )
+    dataset.add(
+        2,
+        StartUIModel.Rockets(
+            tweet = ""
+        )
+    )
+
+    return dataset
```

Comment:
Also you can improve this method using apply extension 
```suggestion
  return mutableListOf<StartUIModel>().apply {
        add(
            StartUIModel.Timer(
                name = nextLaunch.mission.name,
                days = timeToLaunch.day.toString(),
                hours = timeToLaunch.hours.toString(),
                minutes = timeToLaunch.minutes.toString(),
                seconds = timeToLaunch.seconds.toString()
            )
        )
        add(
            StartUIModel.Launches(
                successful = rocketsEfficiency.efficiency,
                total = rocketsEfficiency.total,
                efficiency = "",
                toLaunches = ""
            )
        )
        add(
            StartUIModel.Rockets(
                tweet = ""
            )
        )
    }
```